llvm/test/CodeGen/AArch64/GlobalISel/icmp-trunc.mir

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
The `RUN:` line is missing, I think that is breaking the pre-commit CI.

Reply:
Thanks but this test was added for the discussion with @tschuett and I don't intend to have it in the patch. This change is largely NFC and I am waiting for more reviews to land.

Reply:
I would still suggest adding a RUN line even if you don't intend to include the test because it would make the CI non-red. As a reviewer, seeing green CI gives me more confidence about the patch.

Reply:
I'd rather remove the test. Removed in the latest commit